utilities/common.py

```diff
@@ -155,8 +155,10 @@ def common_parser(for_reco_highPU=False):
     commonargs,_ = parser.parse_known_args()
 
     if for_reco_highPU:
+        if commonargs.sf2DnoUt and commonargs.smooth3dsf:
```

Comment:
Yes, in fact I initially did it like that, but for some reason I noticed that setting commonargs.smooth3dsf=False was apparently not correctly propagating it to the histmakers, maybe because the parser is copied and the original is used. Maybe I had to use the function that updates the defaults, I didn't try at that time

Reply:
I think it might re-parse them when you do parse_args after parse_known_args. It should work if you put this further down with args instead of common_args